### DISCUSSION TITLE
Copilot setup failed

### DISCUSSION THREAD

**Discussion by baojiliu** [community/community#27902] [Copilot Conversations] [answered]
```
Copilot: Server error:^@Please upgrade your Copilot extension to continue using this servic
e.
```

```
nodejs: v16.16.0
neovim: v0.7.2
Copilot.vim:  1.4.0 (git hash: aa9e451dda857c6615f531f8d4e4f201e43d7a03)
```

**Comment by MrSarno** [comment]
> ```
> Copilot: Server error:^@Please upgrade your Copilot extension to continue using this servic
> e.
> ```

Have you tried updating the Copilot extension for Neovim?

> ```
> Copilot.vim:  1.4.0 (git hash: aa9e451dda857c6615f531f8d4e4f201e43d7a03)
> ```

I believe there’s at least a version 1.4.2.

If you use one of the plug-in managers available for Neovim it should be able to update Copilot for you. Otherwise, you can always remove the extension and reinstall it. It’s hard to give step-by-step instructions without knowing more about your setup.

**Comment by baojiliu** [comment, marked as answer]
I switched to Copilot.vim downloaded by the vim ```Plug```
Running the command git pull solved the problem
